apps/api/src/applicant/ienapplicant.util.service.ts

```diff
@@ -151,15 +152,20 @@ export class IENApplicantUtilService {
     applicant: IENApplicant,
     dataToUpdate: any,
     job: IENApplicantJob | null,
-  ): Promise<IENApplicantStatusAudit> {
+  ): Promise<IENApplicantStatusAudit | any> {
```

Comment:
Out of curiosity, why the `any` here

Reply:
was mainly just to test, the return is inside the transaction now, so needed to return something, can change it